tests/rptest/tests/cloud_storage_timing_stress_test.py

```diff
@@ -502,6 +503,7 @@ def start_check(check):
     @parametrize(cleanup_policy="delete")
     @parametrize(cleanup_policy="compact,delete")
     @skip_debug_mode
+    @ok_to_fail
```

Comment:
i was looking at this earlier and it seems like `cleanup_policy="delete"` is the problematic one...annoying that we can't mark a single param as `ok_to_fail`. maybe that's why you put it back to draft though.

Reply:
i put it back in draft because i think we might want to revert a PR that caused this to fail. just testing that theory now